src/test/java/Test4.java

```diff
@@ -0,0 +1,17 @@
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class Test4 {
+
+    @Test
+    public void test(){
+        WebDriver driver = new ChromeDriver();
+        driver.get("https://demoqa.com/automation-practice-form/");
+        WebElement button = driver.findElement(By.tagName("Submit"));
+        System.out.println(button);
```

Comment:
what about setting name and last name?

Reply:
I`m surprised, that I`ve lost that half of the task. Should be more patient